templates/debops__tpl_macros.j2

```diff
@@ -47,6 +47,14 @@
 # ]]] #}
 
 {% macro get_yaml_list_for_elem(list_or_elem) %}{# [[[ #}
+{#
+## Output a YAML formatted list when given a string, boolean or list as argument.
+##
+## Arguments:
+##   list_or_elem: Sting, boolean or list
+##
+## Return: YAML-formatted list
+#}
```

Comment:
This macro has been unified into `flattened()` which is not yet available in your branch. Ref: https://github.com/debops/debops-playbooks/blob/ec9aa0fc8648d28c914a302271171c4ed7fc5f9b/templates/debops__tpl_macros.j2#L133-L192

You can remove the macro completely from this file. Sorry that I forgot to remove it before. All roles which use this legacy macro can be updated over time. (This file is meant as the upstream version as documented at the top of the file. The macro can just be dropped from it).